cannot add prebid.js as a yarn dependency

## Type of issue
<!-- Is this a bug, feature request, question, etc.? Note if this is a documentation issue, please open a ticket here instead: https://github.com/prebid/prebid.github.io -->
Bug

## Description
<!-- Describe the issue -->
When adding prebid.js as a yarn dependency, some modules are not found.

## Steps to reproduce
<!--
  If this is a bug, please provide a list of steps to reproduce the issue and
  if possible a link to a test page or minimal demo of the problem via
  https://jsfiddle.net, http://jsbin.com or similar.
-->
yarn add prebid.js
import prebid from 'prebid.js'

### Expected results
Build success 

### Actual results
Module not found: Can't resolve 'polyfill' in...

## Platform details
<!-- Which versions of Prebid.js, browser / OS, node, npm, etc. -->
prebid: 0.23.1
Browser: Chrome 58.0.3029.110 (64-bit)
OS: OSX El Capitan 10.11.6
node: v7.8.0
yarn: v0.21.3

## Other information
<!-- References to related issue or PR #s, etc. -->

--- a/src/prebid.js
+++ b/src/prebid.js
@@ -4,7 +4,7 @@ import { getGlobal } from './prebidGlobal';
 import { flatten, uniques, isGptPubadsDefined, adUnitsFilter } from './utils';
 import { videoAdUnit, hasNonVideoBidder } from './video';
 import { nativeAdUnit, nativeBidder, hasNonNativeBidder } from './native';
-import 'polyfill';
+import './polyfill';
 import { parse as parseURL, format as formatURL } from './url';
 import { isValidePriceConfig } from './cpmBucketManager';
 import { listenMessagesFromCreative } from './secureCreatives';

